lib/tracing-limit/src/lib.rs

```diff
@@ -55,10 +56,16 @@ where
     pub fn new(layer: L) -> Self {
         RateLimitedLayer {
             events: Default::default(),
+            internal_log_rate_limit: 10,
             inner: layer,
             _subscriber: std::marker::PhantomData,
         }
     }
+
+    pub fn with_limit(mut self, internal_log_rate_limit: u64) -> Self {
```

Comment:
Maybe `with_default_limit` would be a better name since it is overridable at the call site?